src/components/Form/Form.js

```diff
@@ -26,6 +26,7 @@ const Form = ({ handleAddTip }) => {
         // const formattedDate = date.toLocaleString('en-US', { year: 'numeric', month: 'numeric', day: 'numeric' });
 
         handleAddTip({ title: title, description: description, mod: mod, upvotes: 0, date: date, id: Math.random() });
```

Comment:
Can we do object shorthand notation here : `{title, description, mod, upvotes: 0, date, id: Math.random()}`

Reply:
And yes, the shorthand on those keys/properties would be ideal. We can also remove some functions that are using setTitle and setDescription in Form. We can simply use setTitle and setDescription from the return in the Form component.